src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)